src/components/lco/PageBanner.tsx

```diff
@@ -92,56 +104,100 @@ const IndividualBanner: React.FC<ContentProps> = ({ profile }) => (
         </div>
       </div>
     </div>
-  </>
+  </DialogTrigger>
 )
 
 interface ContentProps {
   profile: LCOProfileType
 }
 
-// Previous implementation using hover card
-// const IndividualBanner: React.FC<ContentProps> = ({ profile }) => (
-//  <HoverCard.Root openDelay={300}>
-//    <HoverCard.Trigger>
-//      <ContentTrigger profile={profile} />
-//    </HoverCard.Trigger>
-//    <HoverCard.Portal>
-//      <HoverCard.Content sideOffset={10} side='top' className='z-40'>
-//        <Card profile={profile} />
-//      </HoverCard.Content>
-//    </HoverCard.Portal>
-//  </HoverCard.Root>
-// )
+const LcoCard: React.FC<ContentProps> = ({ profile }) => {
+  const { name, description, email, website, facebook, instagram, report, donation } = profile
+  return (
+
+    <div className='grid md:grid-cols-6 px-11 pt-11 pb-16 gap-2 md:gap-4 bg-white rounded-lg'>
 
-// const ContentTrigger: React.FC<ContentProps> = ({ profile }) => {
-//  const { name, website } = profile
-//  return (
-//    <a
-//      className='block uppercase hover:underline font-medium'
-//      href={website}
-//      target='_blank'
-//      rel='noreferrer'
-//    >
-//      {name}
-//    </a>
-//  )
-// }
+      <div className='md:col-span-1'>
+        <UsersIcon className='h-12 w-12 rounded-lg border-slate-100 border' />
+      </div>
 
-// const Card: React.FC<ContentProps> = ({ profile }) => {
-//  const { name, website, instagram, report, donation } = profile
-//  return (
-//    <div className='card w-96 bg-secondary shadow-xl z-50 overflow-clip border-base-200'>
-//      <div className='p-8 pt-10'>
-//        <h2 className='card-title my-2 uppercase'>{name}</h2>
-//        <div className='flex gap-4'>
-//          <a className='badge badge-outline opacity-60' href={website} target='_blank' rel='noreferrer'>website</a>
-//          <a className='badge badge-outline opacity-60' href={instagram} target='_blank' rel='noreferrer'>instagram</a>
-//          {donation != null && <a className='badge opacity-60 px-4' href={donation} target='_blank' rel='noreferrer'>Donate</a>}
-//        </div>
-//      </div>
-//      <div className='card-actions bg-base-100 flex justify-end p-4'>
-//        <a className='btn btn-primary btn-sm opacity-80' href={report} target='_blank' rel='noreferrer'>Report bad bolts</a>
-//      </div>
-//    </div>
-//  )
-// }
+      <div className='gap-10 md:col-span-5'>
+        <p className='text-base-content/60 font-semibold'>Local Climbing Organization</p>
+        <h2 className='card-title my-2 uppercase'>{name}</h2>
+        <a
+          className='underline pb-5'
+          href={website}
+          target='_blank'
+          rel='noreferrer'
+        >
+          <p className='text-sm'>{website}</p>
+        </a>
+        <p>
+          {description}
```

Comment:
Currently, description data does not have paragraph breaks. I could at the fe parse it to proper form, or, do you think it's more appropriate to clean it in the be? @zichongkao